### ISSUE TITLE
The network does not converge

### ISSUE THREAD

**Issue by fabioperez** [treigerm/WaterNet#4]
Hello,

First of all, thanks for putting this interesting project at GitHub and to make an effort to create a great README file.

I was trying to reproduce your project, so I followed the README and downloaded the same data that you used. After I setup the paths and preprocessed the data, I runned the training with the default values, but the network does not converge even with a huge number of steps (e.g. 5000). I tried to use one-layer and two-layer architectures, and also tried to change the hyperparameters to those one https://github.com/treigerm/WaterNet/tree/trained_models.

Am I missing something?

**Comment by treigerm** [comment]
I think for the model in the repo I used around 9000 training steps. Here's the performance on the validation set from the tensorboard output:
<img width="332" alt="screen shot 2017-02-27 at 14 42 26" src="https://cloud.githubusercontent.com/assets/17100163/23365483/31b8ac2e-fcfb-11e6-8d1f-56628b629e8f.png">

**Comment by fabioperez** [comment]
The validation accuracy is not going up. Also, it's much much lower than I would expect by looking at the image in the README file.

Maybe you should use a different loss / metric [here](https://github.com/treigerm/WaterNet/blob/318616f6405eeb0a009a341c760674a84e163adf/waterNet/model.py#L127).

I'll take a better look at the code.

**Comment by treigerm** [comment]
Yes, there's probably plenty of room to improve the model. My main goal with this project was just to provide an example of analysing geospatial data with neural networks and not to build the best model. Feel free to play around with it and send a pull request if you found a better model!
